crates/sdk/src/action.rs

```diff
@@ -217,4 +227,10 @@ impl<'a> Prove<'a> {
         self.timeout = Some(timeout);
         self
     }
+
+    /// Set the skip deferred proof verification flag.
```

Comment:
can this go in the context builder instead?

Reply:
Moved it.